ActiveRecord.php

```diff
@@ -79,7 +80,7 @@ public static function getDb()
      */
     public static function find()
     {
-        return Yii::createObject(ActiveQuery::className(), [get_called_class()]);
+        return Yii::createObject(ActiveQuery::class, [get_called_class()]);
```

Comment:
Yii2 has supported PHP from 5.4 version.
The constant `class` was released in 5.5.